MAUI/ImageEditor/Crop.md

```diff
@@ -0,0 +1,147 @@
+---
+layout: post
+title: Crop in .NET MAUI ImageEditor control | Syncfusion
+description: Learn here all about crop support in Syncfusion .NET MAUI ImageEditor (SfImageEditor) control.
```

Comment:
description: Learn here all about crop support in the Syncfusion .NET MAUI ImageEditor (SfImageEditor) control.